data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+				die
+			label miss
+			`	You miss your shot. Neither the ship nor the two Hai react, probably not viewing you as a threat. You still have time to do something else or try again. You feel like you heard a chuckle, but you're not sure where it came from.`
+				goto "try again"
+	on defer
+		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
+		log "Accepted a somewhat shady deal for an old Hai ship, an Anomalocaris, that seems to still be in good functioning state."
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Heritage Reclaimed"
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>, with it as your flagship and preferably without any other ships."
+	clearance
+	# Deadline to make sure the player doesn't gain rep for free.
+	deadline 30
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	to offer
+		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
+	to accept
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`One could say that the cargo finds you, rather than you find it, as upon returning to the spaceport, a worker immediately greets you while others begin to fill your ship with crates. A quick check shows have already been allocated for this ship in your name. All that is left to do is to bring it to <destination> by <date> for the promised sum of <payment>.`
+			`	As you head to the bridge, preparing to take off from the planet, you notice a message on the command panel, from your "associates," probably:`
+			`	"Use the Anomalocaris as your flagship.`
+			`	"Do not bring any other ships, as they are not needed for the cargo and would only be a liability at best in Unfettered space, when they could easily provoke them at worst."`
+			`	Given the potential dangers of that area of space, it would be a good idea to familiarize yourself with the design of the Anomalocaris, or perhaps customize it more to your own liking, especially if you plan to deliver the cargo in only this single ship, as requested.`
+				accept
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			branch "wrong flagship"
+				not "flagship model: Anomalocaris"
+			branch "not single ship"
+				"total ships" != 1
+			action
+				"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+				"reputation: Hai (Unfettered)" >?= 0
+			`As you enter the system, you prepare for a fight to get the cargo through, but it is surprisingly quiet. For the first time, entering Unfettered space does not mean getting shot at by missiles from all sides. They're not hailing you either. Perhaps they think you're one of the Hai aiding them, especially given your cargo and the ship you're flying.`
```

Comment:
thats why I used to have a custom swizzle until I was asked to remove it

Reply:
I also removed the bit about them hailing you, since it appears that you can still get hails from the Unfettered during this mission.
```suggestion
			`As you enter the system, you prepare for a fight to get the cargo through, but it is surprisingly quiet. For the first time, entering Unfettered space does not mean getting shot at by missiles from all sides.`
```